### PR TITLE
lambda function enhanced: email functionality and checking works now

### PR DESCRIPTION
**Pull request by anukrr**
Lambda function can now check plants vitals and depending on if warning generated email is sent.
Email has also been enhanced using html.

### CODE REVIEW

File: lambda/check_vitals.py [hunk 1]
```diff
@@ -5,6 +5,29 @@
...
+        print("Error in connection: ", e)
+
+
+connection = get_db_connection()
+
+with connection.cursor() as curr:
+    curr.execute("""USE plants;""")
+    curr.execute("""SELECT * FROM s_alpha.botanist;""")
+    data = curr.fetchall()
+
```
Comment (BenBeard115) [inline lambda/check_vitals.py:30]: Feels strange with this being not in a function or the if __name__ == "__main__" block. Is there a workaround for this?

File: lambda/check_vitals.py [hunk 2]
```diff
@@ -52,39 +73,61 @@ def check_plant_vitals(df: pd.DataFrame) -> list[dict]:
     return unhealthy_plants
 
 
-def send_email():
+
+def send_email(unhealthy_plants:list[dict]):
```
Comment (BenBeard115) [inline lambda/check_vitals.py:77]: Typhint

**Review by BenBeard115 — CHANGES_REQUESTED** [review]
Very nice work, just a few things missing that need to be altered.

**Comment by clv1** [comment]
Stop-gap/old-pull request. Shall I close it instead of merging @anukrr ?

**Pull request closed without merging**